pymc3/distributions/continuous.py

```diff
@@ -2391,7 +2390,16 @@ def logcdf(value, beta):
         )
 
 
-class PyMC3GammaRV(GammaRV):
+class PyMC3GammaRV(RandomVariable):
```

Comment:
To avoid this, we might need to change the original aesara op, see: https://github.com/pymc-devs/pymc3/pull/4548#issuecomment-801750855